docs/release_notes/upgrading_to_destinations_v2.md

```diff
@@ -0,0 +1,170 @@
+1. Existing raw tables replicated to this destination will be copied to a new `airbyte` schema. 
+2. The copied raw table will be updated to the new Destinations V2 format.
+3. The new raw tables will be updated with any new data since the last sync.
+4. The updated raw tables will be typed and de-duplicated according to the Destinations V2 format.
+5. Once typing and de-duplication has completed successfully, your previous final table will be replaced with the updated data.
+
+Pre-existing raw tables, SCD tables and "unnested" tables will always be left untouched. You can delete these at your convenience.
+Each destination version is managed separately, so if you have multiple destinations, they all need to be upgraded one by one.
+
+Versions are tied to the destination. When you update the destination, **all connections tied to that destination will be sending data in the Destinations V2 format**. For upgrade paths that will minimize disruption to existing dashboards, see:
+* [Upgrading Connections One by One with Dual-Writing](#upgrading-connections-one-by-one-with-dual-writing)
+* [Testing Destinations V2 on a Single Connection](#testing-destinations-v2-for-a-single-connection)
+* [Upgrading as a User of Raw Tables](#upgrading-as-a-user-of-raw-tables)
+* [(OSS Only) Rolling back to Legacy Normalization](#oss-only-rolling-back-to-legacy-normalization)
+ 
+## Advanced Upgrade Paths
+
+### Upgrading Connections One by One with Dual-Writing
+
+Dual writing is a method employed during upgrades where new incoming data is written simultaneously to both the old and new systems, facilitating a smooth transition between versions. We recommend this approach for connections where you are especially worried about breaking changes or downtime in downstream systems.
+
+#### Steps to Follow for All Sync Modes
+
+1. **[Open Source]** Update the default destination version for your workspace to a [Destinations V2 compatible version](#destinations-v2-effective-versions). This sets the default version for any newly created destination.
+
+![Upgrade your default destination version](assets/airbyte_version_upgrade.png)
+
+2. Create and configure a new destination connecting to the same database as your existing destination except for `Default Schema`, which you should update to a new value to avoid collisions.
+
+![Create a new destination](assets/airbyte_dual_destinations.png)
+
+3. Create a new connection leveraging your existing source and the newly created destination. Match the settings of your pre-existing connection.
+4. If the streams you are looking to replicate are in **full refresh** mode, enabling the connection will now provide a parallel copy of the data in the updated format for testing. If any of the streams in the connection are in an **incremental** sync mode, follow the steps below before enabling the connection.
+
+#### Additional Steps for Incremental Sync Modes
```

Comment:
Add exceptions for Postgres CDC

Reply:
MySQL CDC may have unknowns.

Reply:
> Does copying the state work (for CDC)?

No. For postgres, this is tricky.  A new CDC Replication Slot will need to be created as well, and the sync will *have* to start from scratch